bentoml/service/env.py

```diff
@@ -178,7 +185,21 @@ def __init__(
         if docker_base_image:
             self._docker_base_image = docker_base_image
         else:
-            self._docker_base_image = config('core').get('default_docker_base_image')
+            if PYTHON_MINOR_VERSION not in PYTHON_SUPPORTED_VERSIONS:
+                logger.warning(
+                    f"Python {PYTHON_VERSION} in current environment is unsupported"
+                    f"the docker image used will contain a different version of Python"
+                    f"supported versions are: f{', '.join(PYTHON_SUPPORTED_VERSIONS)}"
+                )
+                self._docker_base_image = config('core').get(
+                    'default_docker_base_image'
+                )
+            else:
+                self._docker_base_image = (
+                    config('core').get('default_docker_base_image')
```

Comment:
we should be able to get rid of the default_docker_base_image config now, I will follow up with cleanup around this.